english/vote-the-president/trump/WWIII/WWIII.md

```diff
@@ -0,0 +1 @@
+After the wall is completed, President Trump sends Mexico a bill for $8 billion to Mexico. Mexico doesn't like that, and declares war on the U.S. The rest is downhill from here...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.